verify.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, configfile, auth):
             elif opts[line] == '-d':
                 self.desc = opts[line + 1]
             elif opts[line] == '-b':
-                self.bin = opts[line + 1]
+                self.bin = self.desc
```

Comment:
A second caveat: do we want our config object to mirror the import-export tool's config file?  That is, is it clearer in the long run to redefine the config.bin to be a Boolean and all that.  If there were reasons to do that, it might be worth ripping out the config.bin references and changing them to config.desc (or changing everything to config.dir)...